docs/source/tutorials/object_detection.ipynb

```diff
@@ -0,0 +1,501 @@
+    "\n",
+    "An example output of `predictions[i]` is below:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3d70bec6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "predictions[example_idx]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cf95ea28",
+   "metadata": {},
+   "source": [
+    "\n",
+    "Once you have pred_probs and labels in the appropriate formats, you can **find label issues with cleanlab for any object detection dataset**!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3daff923",
+   "metadata": {},
+   "source": [
+    "## 3. Use cleanlab to find label issues\n",
+    "Based on the given `labels` and model `predictions` from our trained model, cleanlab can quickly help us find examples that contain label errors in the dataset. In object detection, label errors are annotations that are more likely imperfect.\n",
+    "\n",
+    "Examples are imperfect when annotators:\n",
+    "\n",
+    "- overlooked an object (missing annotated bounding box),\n",
+    "- chose the wrong class label for an annotated box in the correct location,\n",
+    "- imperfectly annotated the location/edges of a bounding box.\n",
+    "\n",
+    "\n",
+    "For cleanlab, any of these annotation errors should lead to an image with a lower label quality score."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3cceeefb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "num_examples_to_show = 5"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4caa635d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "label_issue_idx = find_label_issues(labels, predictions, return_indices_ranked_by_score=True)\n",
+    "label_issue_idx[:num_examples_to_show]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "638a9dc4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "is_label_issue = find_label_issues(labels, predictions)\n",
+    "is_label_issue[:num_examples_to_show]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5b501dc9",
+   "metadata": {},
+   "source": [
+    "### Get label quality scores\n",
+    "The above example shows how to identify which examples have label quality issues. We can also compute scores for each example in the dataset which estimate our confidence that this example has been correctly labeled using `get_label_quality_scores()`. These scores range between 0 and 1 with smaller values indicating examples whose annotation seems more likely to be imprefect.\n",
+    "\n",
+    "This method is useful for prioritizing which images to review if you have too little time to review all of the label issues by reviewing the images with the lowest scores first."
```

Comment:
```suggestion
    "The above code identifies *which* images have label issues, returning a list of their indices. This is because we specified the `return_indices_ranked_by_score` argument which sorts these indices by the estimated label quality of each image. Below we describe how to directly estimate the label quality scores of each image, but first note that you can omit the `return_indices_ranked_by_score` argument for `find_label_issues()` to instead return a Boolean mask for the entire dataset (True entries in this mask correspond to images with label issues)\n",
\n",
    "### Get label quality scores\n",
    "\n",
    Cleanlab can also compute scores for each image to estimate our confidence that it has been correctly labeled. These label quality scores range between 0 and 1, with *smaller* values indicating examples whose annotation is *more* likely to be wrong in some way.\n",
    "\n",
    "These scores are useful for prioritizing which images to review; if you have too little time, first review the images with the lowest label quality scores."
```